### PR TITLE
chore: update samples for public preview refresh

### PR DESCRIPTION
**Pull request by jebrando**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: digitaltwins/samples/device/javascript/sample_device.js [hunk 1]
```diff
@@ -31,15 +32,20 @@ const propertyUpdateHandler = async (component, propertyName, reportedValue, des
...
   console.log('acknowledgement succeeded.');
 };
 
 
 const environmentalSensor = new EnvironmentalSensor('sensor', propertyUpdateHandler, commandHandler);
 const deviceInformation = new DeviceInformation('deviceInformation');
+const sdkInformation = new SDKInformation('SDKInformation');
 
-const modelId = 'dtmi:YOUR_COMPANY_NAME_HERE:sample_device;1';
+const modelId = 'dtmi:my_company:com:sample_device;1';
```
Comment (anthonyvercolano) [inline digitaltwins/samples/device/javascript/sample_device.js:48]: >my_company [](start = 22, length = 10)

assuming that this looks the same as "c".

File: digitaltwins/samples/device/javascript/sdkInformationInterface.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const BaseInterface = require('azure-iot-digitaltwins-device').BaseInterface;
+const Telemetry = require('azure-iot-digitaltwins-device').Telemetry;
+const Property = require('azure-iot-digitaltwins-device').Property;
+const Command = require('azure-iot-digitaltwins-device').Command;
+
+module.exports.SDKInformation = class SDKInformation extends BaseInterface {
+  constructor(name, propertyCallback, commandCallback) {
+    super(name, 'dtmi:azure:SDKInformation;1', propertyCallback, commandCallback);
+    this.language = new Property();
+    this.version = new Property(true);
```
Comment (anthonyvercolano) [inline digitaltwins/samples/device/javascript/sdkInformationInterface.js:14]: >Property(true) [](start = 23, length = 14)

I don't believe that version and vendor are writable properties.  Passing true here would imply that these are updatable by the service side.

**Comment by anthonyvercolano** [comment]
> 

No sdk information.

---
Refers to: digitaltwins/samples/device/typescript/src/sample_device.ts:98 in e984629. [](commit_id = e98462962cb68ef5e17c29db9e16338ef9cf0545, deletion_comment = False)

File: digitaltwins/device/src/digital_twin_client.ts [hunk 3]
```diff
@@ -231,6 +232,10 @@ export class DigitalTwinClient {
     }, actualCallback);
   }
 
+  getVersion(): string {
+    return packageJson.version
+  }
```
Comment (anthonyvercolano) [inline digitaltwins/device/src/digital_twin_client.ts:237]: Does "c" or Python expose such an API?

File: digitaltwins/samples/device/javascript/sample_device.js (hunk 1 above)
Comment (jebrando) [reply digitaltwins/samples/device/javascript/sample_device.js:48]: Yes sir.

File: digitaltwins/samples/device/javascript/sdkInformationInterface.js (hunk 2 above)
Comment (jebrando) [reply digitaltwins/samples/device/javascript/sdkInformationInterface.js:14]: Once again you are correct.

File: digitaltwins/device/src/digital_twin_client.ts (hunk 3 above)
Comment (jebrando) [reply digitaltwins/device/src/digital_twin_client.ts:237]: C does have a function GetVersionString();

Comment (anthonyvercolano) [reply digitaltwins/device/src/digital_twin_client.ts:237]: You are very correct.  My bad

---
In reply to: [419691523](https://github.com/Azure/azure-iot-sdk-node/pull/779#discussion_r419691523) [](ancestors = 419691523)

File: digitaltwins/device/src/digital_twin_client.ts [hunk 4]
```diff
@@ -115,8 +116,8 @@ export class DigitalTwinClient {
   //
   private _components: {[key: string]: ComponentInformation} = {};
   //
-  // Each Digital Twin client can have only one capability model associated with it.
-  // The dcm is in DTMI format.
+  // Each Digital Twin client can have only one Model ID associated with it.
+  // The ModelID is in DTMI format.
```
Comment (anthonyvercolano) [inline digitaltwins/device/src/digital_twin_client.ts:120]: This comment is simply isolated.  Think it was for the dcm we stored here.  It's no longer at this location.

File: digitaltwins/device/src/digital_twin_client.ts [hunk 5]
```diff
@@ -231,6 +232,10 @@ export class DigitalTwinClient {
     }, actualCallback);
   }
 
+  getVersion(): string {
+    return packageJson.version
```
Comment (anthonyvercolano) [inline digitaltwins/device/src/digital_twin_client.ts:236]: >version [](start = 23, length = 7)

Missing ';'

**Pull request closed without merging**